Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs

```diff
@@ -331,12 +331,25 @@ private ISiteBuilder DetermineProject(string repositoryRoot, string targetPath,
             }
             else if (FunctionAppHelper.LooksLikeFunctionApp())
             {
-                return new FunctionMsbuildBuilder(_environment,
-                                                perDeploymentSettings,
-                                                _propertyProvider,
-                                                repositoryRoot,
-                                                targetPath,
-                                                solutionPath);
```

Comment:
Please avoid unrelated spacing changes to keep the diff minimal. Don't change it this time, but keep in mind for next time. Basically, if you see anything in the diff that doesn't directly related to your PR, it probably shouldn't be included :)